### PR TITLE
HL-10: deployment pipeline configuration

### PR DESCRIPTION
**Pull request by haavikko**
Initial configuration in the .github directory based on the Kesäseteli project. I've never set this up myself so there might be errors :-)

The workflows depend on having the backend/ and frontend/ directories in the repository, which do not yet exist, so the actions would fail with an empty repo. Probably merge this only after we have the backend and frontend skeletons in repo.

### CODE REVIEW

File: .github/workflows/review.yml [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+      - uses: andersinno/kolga-setup-action@v2
+        with:
+          pr_number: ${{ github.event.inputs.pr_number }}
+      - name: Review-Services
+        if: matrix.database
+        uses: andersinno/kolga-deploy-service-action@v2
+        with:
+          projects: ${{ github.event.repository.name }}-${{ matrix.service }}
+        env:
+          POSTGRES_IMAGE: "docker.io/andersinnovations/postgis:11-bitnami"
```
Comment (quyenlq) [inline .github/workflows/review.yml:89]: Maybe switching to Postgres 12 here as we use version 12 for development

**Review by quyenlq — APPROVED** [review]
Looks great! Just a tiny comment about the DB version in `review`. 
We might need maintainer permissions for developers to access this repo setting page, so we can add the missing build env variables. I'm not sure if you already have it @haavikko

File: .github/workflows/review.yml (hunk 1 above)
Comment (tuovinensanttu) [reply .github/workflows/review.yml:89]: The aiven postgress service is configured to use version 11.11 in staging + production. Won't that cause problems?

Comment (quyenlq) [reply .github/workflows/review.yml:89]: Hmn, if so would it make more sense for us to use 11.11 in development too?

Comment (tuovinensanttu) [reply .github/workflows/review.yml:89]: Maybe, at least then there won't be any surprises between local, staging / production. I'm not that familiar with the databases and how well they are backwards compatible.

**Pull request merged**